packages/lesswrong/server/migrations/20221213T181144.fix_float_precision.ts

```diff
@@ -13,6 +13,12 @@
  */
 export const acceptsSchemaHash = "9ff9e6371051f8d49cac5770b07dc0d8";
 
+// The original migration here is commented out as it is incompatible with
+// after adding views to the schema - it was sufficiently long ago that it's no
+// longer needed anyway.
```

Comment:
This migration breaks the postgres bootstrap for new forums since it attempts to change the type of a column that is depended on by a view - we can just comment out the migration now.